core/executor.py

```
@@ -233,6 +567,67 @@ def extract_decisions(self, observations: list) -> list:
+            artifacts: 交付物列表
+        """
+        if self.persistence_manager:
+            try:
+                result = {
+                    "decisions": decisions,
+                    "artifacts": artifacts,
+                    "timestamp": self._get_timestamp()
+                }
+                # 这里可以根据需要实现具体的保存逻辑
+                logger.info(f"Saved execution result: {len(decisions)} decisions, {len(artifacts)} artifacts")
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)